### PR TITLE
fix doc conflict regarding postStart

### PR DESCRIPTION
**Pull request by rajattrt**
Fix the documentation conflict regarding postStart on the page - https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-states

Current observed behavior is documented here -

https://kubernetes.io/docs/tasks/configure-pod-container/attach-handler-lifecycle-event/#discussion
The Container’s status is not set to RUNNING until the postStart handler completes.

Fixes issue #17795

### CODE REVIEW

**Comment by rajattrt** [comment]
signed the CLA.

**Comment by sftim** [comment]
@rajattrt thanks for proposing this! Once Prow can see that you've signed the CLA, we'll be glad to take a look.

**Comment by bradtopol** [comment]
assigning @sftim since he is already engaged on this.
/assign @sftim

**Comment by sftim** [comment]
@rajattrt we can only review (and, I hope, accept) your contribution if you sign a CLA.

Assuming that you're willing to do that&mdash;would you like any help or advice?

**Comment by rajattrt** [comment]
@sftim thanks for the help. Let me try again once.

**Comment by rajattrt** [comment]
Raised PR for this issue - https://github.com/kubernetes/website/pull/18806.
I don't see a cla issue and other checks are in process. 

Thanks all who supported me. Sorry I could not check and fix this earlier.
